### PR TITLE
Adding Owner tab in workloads

### PR DESCRIPTION
**Pull request by urbiz-nr**
See DOC-6840 for more details.

### CODE REVIEW

File: src/content/docs/new-relic-one/use-new-relic-one/workloads/use-workloads.mdx [hunk 1]
```diff
@@ -22,13 +22,22 @@ It comprises the following:
...
 2. Golden metrics. These are charts with the most relevant metrics for each entity type, such as number of requests, response time, and error rate for an application. Explore the charts to detect correlations among different entities (for example, two applications) and different stack layers (for example, applications and hosts). The golden metrics that you see for each entity type on a workload can be customized either at the account or the workload level through the [NerdGraph API](/docs/apis/nerdgraph/examples/golden-metrics-entities-nerdgraph-api-tutorial/#customize-golden-metrics).
 3. Events timeline. This includes the start and end time of incidents and anomalies that refer to the workload entities. It also shows other event types that can explain a change in the status or performance of the workload, such as deployments and configuration changes.    
 
+The **Owner** tab gives you information about the team responsible for the workload. It looks like this:
+
+![workload_owner_tab.png](./images/workloads20210526_owner.png "workloads_owner_tab.png")
+
+It contains:
```
Comment (helenapm) [inline src/content/docs/new-relic-one/use-new-relic-one/workloads/use-workloads.mdx:36]: Thanks @urbiz-nr ! I'm adding some suggestions to guide users a little bit more, a be more explicit about what they can do in the tab. 

It contains:
1. The team that is responsible for the workload. You can include more than one team.
2. The workload description. Share the mission of the workload, and the business logic it represents: Is it a web application? An API? A backend process? Include any details that are relevant to your team, or to other teams in your organization.
3. Contact information. From the drop-down menu, choose how you team prefers to be contacted.
4. Links to the most relevant resources to operate the workload. Here you can add links to runbooks, code repositories, productivity tools, or any other piece of documentation you want at hand concerning the workload.

**Review by helenapm — CHANGES_REQUESTED** [review]
Thanks @urbiz-nr ! I'd mention the option to add the workload Team, and making other suggestions.

File: src/content/docs/new-relic-one/use-new-relic-one/workloads/use-workloads.mdx (hunk 1 above)
Comment (urbiz-nr) [reply src/content/docs/new-relic-one/use-new-relic-one/workloads/use-workloads.mdx:36]: Thanks @helenapm - I'm adding your feedback.

**Review by dbarnesbrownNR — CHANGES_REQUESTED** [review]
I'm all good on peer review from the docs side, but might suggest a small edit to this sentence:

`Here you can add links to runbooks, code repositories, productivity tools, or any other piece of documentation you want at hand concerning the workload.` -> `Here, you can add links to runbooks, code repositories, productivity tools, or anything else related to the workload.`

Mainly suggesting because it makes the flow/grammar easier to parse, but what you have isn't incorrect at all.

Leaving this as `Request changes` because while mine are optional I wasn't sure if you had wanted to get a SME review on this also before publishing, and I see that someone else is added for that. 

Thanks! That's all, looks great otherwise!

**Comment by urbiz-nr** [comment]
Thanks @dbarnesbrownNR, I've updated the doc!

**Review by x8a — APPROVED** [review]

**Review by helenapm — APPROVED** [review]
Awesome!

**Pull request merged**